# State cost matrix Q
state_cost_matrix: [4, 4, 4, 0.8, 0.8, 0.8, 0.0000000001, 0.0000000001, 0.0000000001, 0.0000000001, 0.0000000001, 0.0000000001]

#Motor cost matrix R
#M1 M2 M3 M4 M5 M6 M7 M8
motor_cost_matrix: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
